typescript/src/scss/main.scss

```diff
@@ -1,3 +1,11 @@
+
+  $light_blue: rgb(136, 206, 222);
+  $row_gray: rgb(237, 237, 237);
+  $shading_gray: rgb(222, 222, 222);
```

Comment:
You might be better giving these and `$section_gray` names like `light_bg_gray`, `mid_bg_gray`, `dark_bg_gray` so it's easier to tell which is which later on.